package.json

```diff
@@ -37,11 +38,13 @@
     "d3-dsv": "^1.1.1",
     "d3-scale-chromatic": "^1.3.3",
     "deck.gl": "8.1.0",
+    "higlass": "^1.9.3",
     "json2csv": "^4.5.2",
     "lodash": "^4.17.15",
     "math.gl": "^3.1.3",
     "nebula.gl": "^0.17.4",
     "node-sass": "^4.13.1",
+    "pixi.js": "^5.2.3",
```

Comment:
Does this need to be here because lazy-loading does not load higlass' dependencies?

Reply:
I think it is because pixi is a peer dependency of higlass https://github.com/higlass/higlass/blob/develop/package.json#L41